### PR TITLE
Feature/snapshot deployment

### PR DESCRIPTION
**Pull request by saschadoemer**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
   directories:
   - "~/.m2/repository"
+before_install:
+- chmod +x ./cd/deploy.sh
+after_success:
+- "./cd/deploy.sh"
 language: java
 jdk:
-  - oraclejdk8
-  - oraclejdk9
```
[the remark this answers is not in the document]
Comment (saschadoemer) [reply .travis.yml]: This is not testing, if we place this right here, the artifact is deployed three times for each run. Not the best solution.

File: pom.xml [hunk 2]
```diff
@@ -3,14 +3,48 @@
...
     <modelVersion>4.0.0</modelVersion>
 
-    <name>AGRIROUTER - API Protobuf Definitions</name>
-    <groupId>com.dke.data.api.protobuf</groupId>
+    <groupId>com.agrirouter.proto</groupId>
     <artifactId>definitions</artifactId>
     <version>0.0.1-SNAPSHOT</version>
 
+    <name>AGRIROUTER - API Protobuf Definitions</name>
+    <description>This project contains a simple toolset for working with the agrirouter protobuf files to ensure easy
```
[the remark this answers is not in the document]
Comment (saschadoemer) [reply pom.xml]: Removed the description. Description in README should be all we need.

File: .travis.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Frank-Wiebeler) [reply .travis.yml]: Understood; saw it as a buildpassing-Test. IMHO OK

[the remark this answers is not in the document]
Comment (saschadoemer) [reply .travis.yml]: Then feel free to update the review result 😆

[the remark this answers is not in the document]
Comment (saschadoemer) [reply .travis.yml]: @Frank-Wiebeler Next time feel free to merge 🙊

**Pull request merged**